yarn-error.log

```diff
@@ -0,0 +1,6813 @@
+Arguments: 
```

Comment:
Maybe we should add this file to the .gitignore WDYT?

Reply:
yep